lib/Patat/Images/WezTerm.hs

```diff
@@ -27,6 +35,26 @@ data Config = Config deriving (Eq)
 instance A.FromJSON Config where parseJSON _ = return Config
 
 
+--------------------------------------------------------------------------------
+data Pane =
+    Pane { paneSize     :: Size
+         , paneIsActive :: Bool
+         } deriving (Show, Generic)
+
+instance A.FromJSON Pane where
+    parseJSON = A.genericParseJSON $ AC.aesonPrefix AC.snakeCase
```

Comment:
Since these are just two instances, I would define them manually to avoid the extra `aeson-casing` dependency.